css-text/Overview.bs

```diff
@@ -1296,7 +1299,7 @@ Line Breaking Details</h3>
 
 <pre class='propdef'>
 Name: overflow-wrap, word-wrap
-Value: normal | break-word
+Value: normal | break-word | break-spaces
```

Comment:
thought this was for L4, but I admit that I wasn't paying much attention to which level we resolved on. Have a link to the resolution?